Dockerfile

```diff
@@ -53,133 +91,220 @@ RUN --mount=type=cache,target=/tmp/.gradle/ \
+    && curl -ksSL ${SCANCODE_URL}/v${SCANCODE_VERSION}.tar.gz | tar -C /opt/scancode -xz --strip-components=1 \
+    && cd /opt/scancode \
+    && PYTHON_EXE=python3 ./configure \
+    # Clean up unneeded installed binaries
+    && rm -rf /opt/scancode/thirdparty
+
+#------------------------------------------------------------------------
+# Main container
+FROM eclipse-temurin:11-jre
+
+# ORT
+COPY --from=ortbuild /opt/ort /opt/ort
+COPY docker/ort-wrapper.sh /usr/bin/ort
+COPY docker/ort-wrapper.sh /usr/bin/orth
+RUN chmod 755 /usr/bin/ort
+
+# Python
+COPY --from=pythonbuild /opt/python /opt/python
+COPY --from=pythonbuild /etc/profile.d/python.sh /etc/profile.d/
+
+# Ruby
+COPY --from=rubybuild /opt/rbenv /opt/rbenv
+COPY --from=rubybuild /etc/profile.d/ruby.sh /etc/profile.d/
+
+# NodeJS
+COPY --from=nodebuild /opt/nodejs /opt/nodejs
+COPY --from=nodebuild /etc/profile.d/nodejs.sh /etc/profile.d/
+
+# Golang
+COPY --from=gobuild /opt/go /opt/go/
+COPY --from=gobuild  /etc/profile.d/go.sh /etc/profile.d/
+
+# Haskell
+COPY --from=haskellbuild /usr/bin/stack /usr/bin
+
+# Repo and Android
+COPY --from=androidbuild /usr/bin/repo /usr/bin/
+COPY --from=androidbuild /opt/android-sdk /opt/android-sdk
+COPY --from=androidbuild /etc/profile.d/android.sh /etc/profile.d/
+
+# ScanCode
+COPY --from=scancodebuild /opt/scancode /opt/scancode
+RUN ln -s /opt/scancode/bin/scancode /usr/bin/scancode \
+    && ln -s /opt/scancode/bin/pip /usr/bin/scancode-pip \
+    && ln -s /opt/scancode/bin/extractcode /usr/bin/extractcode
+
+RUN  --mount=type=cache,target=/var/cache/apt --mount=type=cache,target=/var/lib/apt apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+        ca-certificates \
+        cvs \
+        curl \
+        gnupg \
+        libarchive-tools \
         netbase \
-        openssh-client \
-        unzip \
-        xz-utils \
-        zlib1g-dev \
-        # Install VCS tools (no specific versions required here).
+        openssl \
+        unzip
+
+# External repositories for SBT
+ARG SBT_VERSION=1.3.8
+ENV KEYURL=https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823
+RUN echo "deb https://repo.scala-sbt.org/scalasbt/debian all main" | tee /etc/apt/sources.list.d/sbt.list
+RUN echo "deb https://repo.scala-sbt.org/scalasbt/debian /" | tee /etc/apt/sources.list.d/sbt_old.list
+RUN curl -ksS "$KEYURL" | gpg --dearmor | tee "/etc/apt/trusted.gpg.d/scala_ubuntu.gpg" > /dev/null
+
+# External repository for latest Git
+ENV KEYURL=https://keyserver.ubuntu.com/pks/lookup?op=get&search=0xa1715d88e1df1f24
+RUN echo "deb http://ppa.launchpad.net/git-core/ppa/ubuntu bionic main" > /etc/apt/sources.list.d/git-core-ubuntu-ppa-bionic.list
+RUN curl -ksS "$KEYURL" | gpg --dearmor | tee "/etc/apt/trusted.gpg.d/git-core_ubuntu_ppa.gpg"  > /dev/null
+
+ENV COMPOSER_VERSION=1.10.1-1 \
+    CONAN_VERSION=1.40.3
+
+ENV PATH="$PATH:$HOME/.local/bin"
```

Comment:
If someone runs the container as other user, the directory `$HOME/.local/bin` does no longer point to the right folder or points to a folder that is not readable.

Reply:
HOME seems to be unset on build time:
```
$ docker run -ti --entrypoint=/bin/bash ort
root@1cc91553aeff:/# echo $PATH
/opt/java/openjdk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/.local/bin
```

Reply:
Actually, no .local/bin is needed anymore, we don't have any tool there anymore. 
 I just forgot to remove it

Reply:
Solved